.pre-commit-config.yaml

```diff
@@ -13,3 +13,11 @@ repos:
       - id: black
         args:
         - --preview
+
+  - repo: https://github.com/pycqa/flake8
+    rev: 6.1.0
```

Comment:
What does this rev refer to? Our latest tag / release is 23.9.16

If we can not use it cause re remove it or make it 99.0.0 or something + comment why we've done so